state/model.go

```diff
@@ -95,6 +95,12 @@ type modelEntityRefsDoc struct {
 
 	// Applicatons contains the names of the applications in the model.
 	Applications []string `bson:"applications"`
+
+	// Volumes contains the IDs of the volumes in the model.
```

Comment:
perhaps s/in the model/bound to the model
to be consistent with terminology used elsewhere?